dist/index.html

```diff
@@ -1,17 +1,42 @@
 <!DOCTYPE html>
 
 <html lang="en">
+  
   <head>
-    <meta charset="utf-8" />
-
-    <title>SHOT RUN</title>
-    <meta name="description" content="SHOT RUN" />
+    <meta charset="UTF-8" />
+    <!-- Update title to "Sober Run" not "Shot Run"-->
+    <title>Sober Run</title>
+    <meta name="description" content="Web Browser Game" />
     <meta name="author" content="Daniel C. Suh and Jaehoon Han" />
+    <!--add a viewport meta tag-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    
+    <style>
```

Comment:
lets move the styles to a separate CSS file and just import it in this file.

usually, devs these days like to uphold a "separate of concerns" with files (have a file do 1 thing). in this case, we want to separate HTML from styling. they're 2 different concerns